plugins/processors/math/math.go

```diff
@@ -0,0 +1,145 @@
+package math
+
+import (
+	"fmt"
+	"log"
+	"math"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+// Math function congfig
+type Math struct {
+	Func   string   `toml:"function"`
+	Metric string   `toml:"measurement_name"`
+	Fields []string `toml:"fields"`
```

Comment:
might want to also take a float64 argument variable, which would let you support the entire _math_ package, as well as addition, subtraction, multiplication, and division